### PR TITLE
GenreCell: Added Cell to display genres

### PR DESCRIPTION
**Pull request by carolanitz**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: SharedSources/MediaLibraryModel/MediaModel.swift [hunk 1]
```diff
@@ -61,3 +61,13 @@ extension VLCMLMedia {
         return integer == 0
     }
 }
+
+extension VLCMLGenre {
+    @objc func numberOfTracksString() -> String {
```
Comment (Mikanbu) [inline SharedSources/MediaLibraryModel/MediaModel.swift]: This should be inside `GenreModel` and not inside `MediaModel`.

**Comment by carolanitz** [comment]
@Mikanbu yes the image can be changed later and you can see the results already. I also moved the  numberOfTracksString into GenreModel. Nice catch!

File: Sources/MediaCategoryCells/GenreCollectionViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    @objc fileprivate func themeDidChange() {
+        backgroundColor = PresentationTheme.current.colors.background
+        titleLabel.textColor = PresentationTheme.current.colors.cellTextColor
+        numberOfTracksLabel.textColor = PresentationTheme.current.colors.cellDetailTextColor
+    }
+
+    func update(genre: VLCMLGenre) {
+        titleLabel.text = genre.name
+        numberOfTracksLabel.text = genre.numberOfTracksString()
+//        if genre.isThumbnailGenerated() {
```
Comment (Mikanbu) [inline Sources/MediaCategoryCells/GenreCollectionViewCell.swift]: I think this comment can be removed for now since Genre doesn't have thumbnails.

**Comment by carolanitz** [comment]
removed the commented out code for thumbnail, wrote ticket, opened https://code.videolan.org/videolan/vlc-ios/issues/380 for followup work and merged with 7aadb92e9b8efca4d6d19c27886dbbd7df71794e

**Pull request closed without merging**